cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -201,6 +207,30 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
             services[it.containerName] = service
         }
 
+        if (external.containerImage.isNotEmpty()) {
+            val extra = mutableMapOf(
+                    "image" to external.containerImage,
+                    "ports" to external.servicePorts.toList().map{QuotedString("${it}:${it}")},
+                    "expose" to external.servicePorts
+            )
```

Comment:
You need to invoke `get()` to retrieve a lazy property's value. And I think the `toList()` is unnecessary. So this would become:
```kotlin
val extra = mutableMapOf(
    "image" to external.containerImage.get(),
    "ports" to external.servicePorts.get().map {QuotedString("${it}:${it}") },
    "expose" to external.servicePorts.get()
)
```